buildSrc/src/main/java/Dependencies.kt

```diff
@@ -45,23 +49,14 @@ object Dependencies {
     const val multidex = "androidx.multidex:multidex:2.0.1"
     const val datastorePrefs = "androidx.datastore:datastore-preferences:1.0.0-beta01"
 
-    object Hilt {
-      const val dagger = "com.google.dagger:hilt-android:$DAGGER_HILT_VERSION"
-      const val daggerCompiler = "com.google.dagger:hilt-compiler:$DAGGER_HILT_VERSION"
-      const val daggerHiltCompiler = "androidx.hilt:hilt-compiler:$ANDROIDX_HILT_VERSION"
-      const val hiltLifecycleViewModel =
-          "androidx.hilt:hilt-lifecycle-viewmodel:$ANDROIDX_HILT_VERSION"
-      const val hiltNavigation = "androidx.hilt:hilt-navigation:$ANDROIDX_HILT_VERSION"
-      const val hiltNavigationFragment = "androidx.hilt:hilt-navigation-fragment:1.0.0-beta01"
```

Comment:
Hilt is not a part of AndroidX but rather a Google library, so moved to Google block